### ISSUE TITLE
Should cookie_session_no_expires rule recognise maxAge property?

### ISSUE THREAD

**Issue by fierydrake** [ajinabraham/njsscan#40]
According to the express-session documentation session configuration should not specify `expires` directly but prefer `maxAge`:

> 
> cookie.expires
> 
> ...
> 
> **Note** The expires option should not be set directly; instead only use the maxAge option.

Source: https://www.npmjs.com/package/express-session#cookieexpires

With that in mind should the rule be updated to check for `maxAge` instead of `expires` or else recognise `maxAge` as an alternative to `expires`.

In the mean time, is there a good reason why I should not ignore failure of this rule if I have set `maxAge`?

**Comment by ajinabraham** [comment]
Thanks for the heads up, I will take a look. 

Regarding the last part, if your application targets older browsers like some versions of Internet Explorer that do not support maxAge, your cookie may never expire.

**Comment by fierydrake** [comment]
From what I can tell express-session calculates a value for `expires` based on the `maxAge` configuration option rather than setting `Max-Age` in the `Set-Cookie` header directly. This is based on observation rather than code inspection though.

For example, I have not set the `expires` in the express-session options, only `maxAge` and this results in a `Set-Cookie` header similar to this one:
```
Set-Cookie: sessionId=s%3AZiIJEDetcnA9F7LT_hUPYYm5hLA7wmLE.tEUDjI8GwXjMvNF1xgZ1b8JyF2XsY6SCMcV5RNKqD2I; Domain=localhost; Path=/; Expires=Fri, 18 Dec 2020 03:05:47 GMT; HttpOnly; SameSite=Strict
```

**Comment by ajinabraham** [comment]
Looks like to be backward compatible, maxAge gets converted to expires https://github.com/expressjs/session/blob/a8641429502fcc076c4b2dcbd6b2320891c1650c/session/cookie.js#L92

**Comment by ajinabraham** [comment]
Addressed in 0.2.2
